CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 Changelog
 =========
 
+## 1.4.0 (2024-01-25)
+
+### Enhancements
+
+* swizzleViewLoadPreMain setting is now false by default. When this config value is true, all custom view controllers are swizzled at app start, which may delay app start if there are many of them.
+  [202](https://github.com/bugsnag/bugsnag-cocoa-performance/pull/202)
+
+### Bug fixes
+
+* Detect pre-loaded views and correct their durations.
```

Comment:
This one could probably benefit from some description and also moving to the Enhancements section:

```suggestion
* Detect pre-loaded views and correct their durations. These can occur in container views, such as `UITabViewController` when the view is loaded in anticipation of the user opening the next tab. The resulting spans are now marked with a "pre-loaded" suffix and the timings adjusted to start from `viewWillAppear` only. 
```